src/components/forms/SelectField/SelectField.js

```diff
@@ -68,6 +85,14 @@ const SelectField = ({
           />
         )}
         {...props}
+        onChange={newValue => {
+          const isSingle = !Array.isArray(newValue);
+          const normalized = isSingle ? [newValue] : newValue;
+          const rawValues = normalized.map(val => val?.[valueProperty] ?? null);
```

Comment:
The `?? null` turns `undefined` into `null`?

Reply:
Yep, while leaving other falsy values like `''` and `0` untouched.